packages/extism-tools/examples/tool/wiki.js

```diff
@@ -0,0 +1,12 @@
+import { OpenAIAgent } from "llamaindex";
+import { ExtismToolFactory } from "../../dist/ExtismToolFactory.js";
```

Comment:
goal should be to use `exitism-tools` as package, so add a package.json to the examples folder that is using this package